src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-710.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2021-05-28'
 version: 7.1.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-7.1.0.gem'
+features: ["Browser agent can run on applications using CSP nonces", "Distributed tracing may now be set in server-side application configuration"]
+bugs: ["Update conflicts with use of Module#Prepend", "Update support for ActiveRecord 6.1+ instrumentation"]
```

Comment:
Is there anything more specific about what was fixed for Active Record 6.1?

Reply:
[Added](https://github.com/newrelic/docs-website/pull/11703/commits/f755804b3a1fea8abcd87d362cc92a4c34c3ea03#diff-5f7d5b8a5623e491aed8f3ad2eddd73aa0afe59f1946332c0a0166c9ae8cd5f8R7) more info for this!